tests/modules/getchannels.c

```diff
@@ -0,0 +1,66 @@
+#include "redismodule.h"
+#include <strings.h>
+#include <assert.h>
+#include <unistd.h>
+#include <errno.h>
+
+/* A sample with declarable channels, that are used to validate against ACLs */
+int getChannels_subscribe(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+    if ((argc - 1) % 3 != 0) {
+        RedisModule_WrongArity(ctx);
+        return REDISMODULE_OK;
+    }
+    char *err = NULL;
+    /* Handle getkeys-channels introspection */
```

Comment:
```suggestion
    /* Handle getchannels-api introspection */
```
actually, i'm not sure that comment is right.
maybe instead describe here the command syntax for this command?

Reply:
Yeah, I don't know what I meant here.